slatedb/src/db.rs

```diff
@@ -924,22 +945,18 @@ impl Db {
         }
     }
 
-    /// TODO: this #[allow(unused)] will be removed soon in #519.
     #[allow(unused)]
```

Comment:
Should the allow unused be removed as well?

Reply:
i failed to remove this method immediately for it's still referenced in tests, i changed `#[unused]` into `#[cfg(test)]` in bb06c78, but i suppose there might have possible way to remove this depenency in tests, let me have a try about it 🤔